### PR TITLE
[debug][tao bridge] dump cluster input proto in debug mode

### PR DESCRIPTION
**Pull request by wyzero**

### CODE REVIEW

File: tao/tao_bridge/kernels/tao_compilation_cache.cc [hunk 1]
```diff
@@ -333,6 +351,11 @@ TaoCompilationCache::TaoCompilationCache(bool async_compilation)
   graphdef_node_size_min_ = dumper_opts->graphdef_node_size_min;
   graphdef_node_size_max_ = dumper_opts->graphdef_node_size_max;
 
+  if (opts->disc_debug_mode) {
```
Comment (Yancey1989) [inline tao/tao_bridge/kernels/tao_compilation_cache.cc:354]: Can we specify a fixed iteration number with `disc_debug_mode` ?

Comment (wyzero) [reply tao/tao_bridge/kernels/tao_compilation_cache.cc:354]: Personally, I think the meaning of 'fixed iteration number' is not well defined here. The compilation cache will trigger the compilation process when the rank of the cluster is changed. In most cases, we only need to compile once.

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Comment by Yancey1989** [comment]
Maybe we can add a unit test in the future.

**Pull request merged**